manifest.json

```diff
@@ -1,27 +1,32 @@
 {
     "name": "__MSG_extensionName__",
     "author": "Talis Education Limited",
-    "description":"__MSG_description__",
+    "description": "__MSG_description__",
     "version": "__VERSION__",
-    "icons":  {
+    "icons": {
         "16": "images/icon16.png",
         "48": "images/icon48.png",
         "128": "images/icon128.png",
         "176": "images/icon176.png"
     },
     "background": {
-        "page": "background.html",
-        "persistent": false
+        "service_worker": "background.js",
```

Comment:
This is probably the biggest departure. Extensions only work as `service workers` and not background scripts. I _think_ this will necessitate us reworking what we have.